src/plugins/reporter-systemd-journal.c

```diff
@@ -308,7 +308,7 @@ int main(int argc, char **argv)
         problem_data_add_text_noteditable(problem_data, "crash_function", "??");
 
     /* Add SYSLOG_IDENTIFIER into problem data */
-    if (syslog_id)
+    if (syslog_id || (syslog_id = getenv("reporter_journal_syslog_id")))
```

Comment:
Is there any reason to use lowercase name? I think all libreport tools accept uppercase environment variables.